CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased (v7)
+* Breaking Changes
+  * Bump minimum supported deployment target to iOS 16+
+* BraintreeVenmo
+  * Update `BTVenmoRequest` to make all properties accessible on the initializer only vs via the dot syntax.
+
 ## unreleased
 * BraintreePayPal
   * Add `BTPayPalRequest.userPhoneNumber` optional property
+* BraintreeVenmo
+  * Send `url` in `event_params` for App Switch events to PayPal's analytics service (FPTI)
```

Comment:
Should this diff be in this PR?

Reply:
ahh good catch! it shouldn't, updated in c386bf3f3eca0db14967a3da1611e2c0f6310230